src/transformers/models/moshi/configuration_moshi.py

```diff
@@ -0,0 +1,229 @@
+            The standard deviation of the truncated_normal_initializer for initializing all weight matrices.
+        use_cache (`bool`, *optional*, defaults to `True`):
+            Whether or not the model should return the last key/values attentions (not used by all models). Only
+            relevant if `config.is_decoder=True`.
+        sliding_window (`int`, *optional*, defaults to 3000):
+            Sliding window attention window size. If not specified, will default to `3000`.
+        attention_dropout (`float`, *optional*, defaults to 0.0):
+            The dropout ratio for the attention probabilities.
+        ffn_dim (`int`, *optional*, defaults to 22528):
+            Dimensionality of the "intermediate" (often named feed-forward) layer in the main decoder block. Must be even.
+        rms_norm_eps (`float`, *optional*, defaults to 1e-08):
+            The epsilon used by the rms normalization layers.
+        num_codebooks (`int`, *optional*, defaults to 8):
+            The number of audio codebooks for each audio channels.
+        depth_hidden_size (`int`, *optional*, defaults to 1024):
+            Dimensionality of the layers and the pooler layer of the depth decoder.
+        depth_num_hidden_layers (`int`, *optional*, defaults to 6):
+            Number of depth decoder layers.
+        depth_max_position_embeddings (`int`, *optional*, defaults to 9):
+            The maximum sequence length that the depth decoder model might ever be used with. Typically, set this to the
+            number of codebooks.
+        depth_num_attention_heads (`int`, *optional*, defaults to 16):
+            Number of attention heads for each attention layer in the depth decoder block.
+        depth_ffn_dim (`int`, *optional*, defaults to 5632):
+            Dimensionality of the "intermediate" (often named feed-forward) layer in the depth decoder block. Must be even.
+        depth_head_dim (`int`, *optional*, defaults to `depth_hidden_size // depth_num_attention_heads`):
+            The attention head dimension of the depth encoder layers.
+        depth_num_key_value_heads (`int`, *optional*):
+            This is the number of key_value heads that should be used to implement Grouped Query Attention in the depth decoder.
+            If it is not specified, will default to `depth_num_key_value_heads`.
+        depth_sliding_window (`int`, *optional*, defaults to 8):
+            Sliding window attention window size. If not specified, will default to `8`.
+            Whether input and output word embeddings should be tied.
+        tie_word_embeddings (`bool`, *optional*, defaults to `False`):
+            Whether to tie weight embeddings
+        kwargs (*optional*):
+            Dictionary of keyword arguments. Notably:
+                - **audio_encoder** ([`PretrainedConfig`], *optional*) -- An instance of a configuration object that
+                  defines the audio encoder config.
```

Comment:
that is a bit weird, we usually bass `audio_encoder_config` but this is more convenient for you i guess?

Reply:
I can refactor, no problems